src/components/drawer-content/drawer-content.tsx

```diff
@@ -96,6 +121,107 @@ const DrawerContent: React.FC<Props> = () => {
     );
   }
 
+  /**
+   * Renders the Total work time section
+   * 
+   * @param name name of the row
+   * @param value value of the row
+   * @param color color of the row value
+   */
+  const renderAccordinRow = (name: string, value: string, color?: string) => {
+
+    return (
+      <Box className={ classes.accordinRow }>
+        <Typography style={{ fontSize: 14, fontWeight: 600 }}>
+          { name }
+        </Typography>
+        <Typography style={{ fontSize: 14, fontStyle: "italic", color: color }}>
+          { value }
+        </Typography>
+      </Box>
+    );
+  }
+
+  /**
+   * Renders the Total work time section
+   */
+  const renderTotalWorkTime = () => {
+    if (!personTotalTime) {
+      return null;
+    }
+
+    let totalHour = TimeUtils.minuteToHourString(personTotalTime.total);
+    personTotalTime.total >= 0 && (totalHour = `+${totalHour}`)
```

Comment:
How does this line modify the totalHour and is it needed?

Reply:
Well, the converter converts the time to (-)x h y min, the negative time is indicated with a minus sign but the positive is not. It is used for the expected time and logged time which is always positive but the total time could be positive/ negative. So an extra plus is there for total time to keep it in a format of (+/-) x h y min

Reply:
I see, it can be left as is then.